doc/faq.rst

```diff
@@ -396,3 +396,38 @@ and not at test time, for resampling and similar uses,
 like at `imbalanced-learn`.
 In general, these use cases can be solved
 with a custom meta estimator rather than a Pipeline
+
+Why are there so many different estimators for linear models?
+-------------------------------------------------------------
+Usually, there is one classifier and one regressor per algorithms, e.g.
```

Comment:
You probably mean "model", rather than "algorithm". An algorithm eg is the type of solver used.

Reply:
How about "model type"?